plugins/simpleadminhacks-paper/src/main/resources/config.yml

```diff
@@ -516,4 +516,32 @@ hacks:
     enabled: false
     cooldownTime: 100
   DespawnFix:
-    enabled: true
\ No newline at end of file
+    enabled: true
+  BetterRails:
+    enabled: true
+    # All in metres per second
+    # Minecraft will prevent you going faster than 30m/s
+    base: 12
+    materials:
+      COBBLESTONE: 8
+      COPPER_BLOCK: 29
+      EXPOSED_COPPER: 23
+      WEATHERED_COPPER: 18
+      OXIDIZED_COPPER: 14
+    skyBase: 1
+    skyMaterials:
+      COBBLESTONE: 0
+      COPPER_BLOCK: 1
+      EXPOSED_COPPER: 1
+      WEATHERED_COPPER: 1
+      OXIDIZED_COPPER: 1
```

Comment:
This config section needs some work, as it stands the config options are confusing and it's not clear that `skyBase` is added to `base`, and lacks the ability to disable subfeatures.

I would arrange it like this:

```yaml
  BetterRails:
    enabled: true
    baseSpeed: 12
    materials:
      COBBLESTONE: 8
      COPPER_BLOCK: 29
      EXPOSED_COPPER: 23
      WEATHERED_COPPER: 18
      OXIDIZED_COPPER: 14
    skyBonus:
      enabled: true
      bonus: 1
```

If sky materials are really needed to be different,

```yaml
  BetterRails:
    enabled: true
    speeds:
      - type: default
         baseSpeed: 12
         materials:
           COBBLESTONE: 8
           COPPER_BLOCK: 29
           EXPOSED_COPPER: 23
           WEATHERED_COPPER: 18
           OXIDIZED_COPPER: 14
      - type: sky
         baseSpeed: 13
         materials:
           COBBLESTONE: 9
           COPPER_BLOCK: 30
           EXPOSED_COPPER: 24
           WEATHERED_COPPER: 19
           OXIDIZED_COPPER: 15
```